### ISSUE TITLE
Typescript Type Definition

### ISSUE THREAD

**Issue by ionizer** [lovell/farmhash#32]
Hi there,

I was trying to use this package for Typescript but it's giving me an error saying that the type declaration (.d.ts) file doesn't exist.
```
Could not find a declaration file for module 'farmhash'. '.../node_modules/farmhash/index.js' implicitly has an 'any' type.
  Try `npm install @types/farmhash` if it exists or add a new declaration (.d.ts) file containing `declare module 'farmhash';`
```

Can we add the declaration file for this module? 

I think I can help with a PR for this, or actually I'm not really sure if we should publish it to @types repo (and sadly I don't really know the conventions for this yet). If not, `dts-gen` package generates the following .d.ts file:
```
/** Declaration file generated by dts-gen */

export function fingerprint32(input: any): any;

export function fingerprint64(input: any): any;

export function hash32(input: any): any;

export function hash32WithSeed(input: any, seed: any): any;

export function hash64(input: any): any;

export function hash64WithSeed(input: any, seed: any): any;

export function hash64WithSeeds(input: any, seed1: any, seed2: any): any;
```

P.S.: Using `dts-gen` to generate the above file and putting it on my `types` directory works for Typescript projects. But I think it's still better if we don't need to use `dts-gen` 😁

**Comment by lovell** [comment]
Hi, happy to accept a PR with an `index.d.ts` if you're able - we should probably use more specific types than `any` if possible.

**Comment by ionizer** [comment]
I have added a more specific .d.ts to my fork and created a PR. Seems to work (at least with VSCode) when I try to use the package directly from my fork by putting the following in package.json's dependencies:
```
"farmhash": "git+https://github.com/ionizer/farmhash.git",
```

**Comment by lovell** [comment]
Thanks for the definitions file, published in v3.1.0.
